tests/test_config_flag.py

```diff
@@ -0,0 +1,16 @@
+"""Check that the --config command-line flag works"""
```

Comment:
The test name and docstring suggest that this is testing the `--config` option, but the body is testing the `load_options` functions, which is one part of the pipeline that underlies the `--config` option handling. Did you test the end-to-end pipeline using the CLI tool? Minimally you could confirm you tested it manually. More fully, you could include a test of the `proselint()` function (setting the `config` kwarg) or even the `proselint` CLI command with the `--config` option set and not set.

Reply:
Also, we're not using `--config` as a "flag" here, but as an "option".